samples/vsphere/sso/external_psc_sso_workflow.py

```diff
@@ -82,7 +82,8 @@ def options(self):
         self.args = self.argparser.parse_args()
 
     def setup(self):
-        self.lswsdl = self.args.lswsdl
```

Comment:
I agree.. Second IF condition was to check if lswsdl is None always. I think else is makes more sense.
But since lswsdl is instance variable, skipping self.lswsdl = self.args.lswsdl, adding check on arg passed